opentelemetry-sdk/CHANGELOG.md

```diff
@@ -2,12 +2,14 @@
 
 ## Unreleased
 
-- Rename `MetricRecord` class to `ExportRecord`
-  ([#1367](https://github.com/open-telemetry/opentelemetry-python/pull/1367))
 - Add optional parameter to `record_exception` method
   ([#1314](https://github.com/open-telemetry/opentelemetry-python/pull/1314))
+- Allow samplers to modify tracestate
+  ([#1319](https://github.com/open-telemetry/opentelemetry-python/pull/1319))
 - Update exception handling optional parameters, add escaped attribute to record_exception
   ([#1365](https://github.com/open-telemetry/opentelemetry-python/pull/1365))
+- Rename `MetricRecord` class to `ExportRecord`
+  ([#1367](https://github.com/open-telemetry/opentelemetry-python/pull/1367))
 - Rename Record in Metrics SDK to Accumulation
   ([#1373](https://github.com/open-telemetry/opentelemetry-python/pull/1373))
```

Comment:
When I look at [the git history], if we care about the order in which items were merged, I think the ordering should look like this (from oldest to newest):

```suggestion
- Allow samplers to modify tracestate
  ([#1319](https://github.com/open-telemetry/opentelemetry-python/pull/1319))
- Add optional parameter to `record_exception` method
  ([#1314](https://github.com/open-telemetry/opentelemetry-python/pull/1314))
- Update exception handling optional parameters, add escaped attribute to record_exception
  ([#1365](https://github.com/open-telemetry/opentelemetry-python/pull/1365))
- Rename `MetricRecord` class to `ExportRecord`
  ([#1367](https://github.com/open-telemetry/opentelemetry-python/pull/1367))
- Rename Record in Metrics SDK to Accumulation
  ([#1373](https://github.com/open-telemetry/opentelemetry-python/pull/1373))
- Rename Meter class to Accumulator in Metrics SDK
  ([#1372](https://github.com/open-telemetry/opentelemetry-python/pull/1372))
```

**PLEASE NOTE:** I couldn't reach the git suggestion all the way to the bottom, I thought it would be easiest to have it deleted and just replace it with the suggestion above